scripts/test_example_projects_ci.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+SCRIPT_LOCATION=$(dirname -- "$(readlink -f -- "${BASH_SOURCE[0]}")")
+
+source ${SCRIPT_LOCATION}/helper_functions.sh
+
+set -e
+
+if [ ! -e 'gradle.properties' ]; then
+    echo "Script was not executed in root directory"
+    exit 1
+fi
+
+
+VERSION="1.0-t"
+
+echo Updating version in gradle.properties...
+for file in "gradle.properties" \
+"example-projects/junit5/gradle.properties" \
+"example-projects/spock/gradle.properties" \
+"example-projects/testng/gradle.properties" \
+"example-projects/android/gradle.properties" \
+"example-projects/selenium/gradle.properties" \
+"example-projects/spring-boot/gradle.properties" \
+"jgiven-gradle-plugin/gradle.properties"
+do
+  update_version "${VERSION}" "${file}" || exit 1
+done
+
+#Publishing to mavenLocal
+./gradlew clean publishToMavenLocal
+
+
+echo "Testing the example projects that use the Maven plugin..."
+
+for file in "example-projects/java11/pom.xml" \
+"example-projects/maven/pom.xml"
+do
+  mvn -U -f ${file} clean test -Djgiven.version=$VERSION
+done
+
+echo "Testing the example projects that use the Gradle plugin..."
+
+for file in "example-projects/junit5/build.gradle" \
```

Comment:
It might be nice to have each of these cases be a separate test step, so that you immedeately see, which one failed.